api4/user.go

```diff
@@ -1131,6 +1151,37 @@ func updateUser(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(ruser.ToJson()))
 }
 
+// checkProviderAttributes returns the empty string if the patch can be applied without
+// overriding attributes set by the user's login provider; otherwise, the name of the offending
+// field is returned.
+func checkProviderAttributes(a app.AppIface, ouser *model.User, patch *model.UserPatch) string {
+	tryingToChange := func(patchValue *string, userValue *string) bool {
```

Comment:
nit: lets change the order here to match the function to give the idea of `trying to change users value to patch value`.
```suggestion
	tryingToChange := func(userValue *string, patchValue *string) bool {
```